### DISCUSSION TITLE
Linaria

### DISCUSSION THREAD

**Discussion by NikolaySolodukhin** [ben-rogerson/twin.macro#487] [Ideas]
Hi! Do you have any plans to add support linaria? 

Why Linaria? Linaria is the same styled-components but with Zero-runtime CSS in JS library. Linaria is based on css-variables

**Comment by ben-rogerson** [comment]
Hey there, yes there are plans to add linaria once [linaria@v3](https://github.com/callstack/linaria/releases) is off beta and stabilised.

**Reply by vinverdy** [reply]
linaria already v4 rn

**Comment by hotrungnhan** [comment]
+1 is there a hugh improve in performance when using linaria ?

**Comment by judehunter** [comment]
Rooting for this to land!
